ethernet_init/idf_component.yml

```diff
@@ -1,4 +1,7 @@
 dependencies:
```

Comment:
Please update the minor version of the component.

Reply:
Done.